dev/NavigationView/NavigationView_rs1_themeresources.xaml

```diff
@@ -163,10 +163,10 @@
         </ResourceDictionary>
 
         <ResourceDictionary x:Key="HighContrast">
-            <StaticResource x:Key="NavigationViewDefaultPaneBackground" ResourceKey="SystemChromeMediumColor" />
-            <StaticResource x:Key="NavigationViewExpandedPaneBackground" ResourceKey="SystemColorWindowColor" />
-            <StaticResource x:Key="NavigationViewTopPaneBackground" ResourceKey="SystemChromeMediumHighColor" />
-            <StaticResource x:Key="NavigationViewContentBackground" ResourceKey="SystemChromeMediumColor" />
+            <SolidColorBrush x:Key="NavigationViewDefaultPaneBackground" Color="{ThemeResource SystemColorWindowColor}" />
```

Comment:
Unfortunate change from ChromeMedium to WindowColor for all the **navView parts in HC** but according to accessibility requirements we have to do that :(